specification/redis/Cache.Management/back-compatible.tsp

```diff
@@ -0,0 +1,232 @@
+  "preferredDataPersistenceAuthMethod"
+);
+@@clientName(RedisCommonPropertiesRedisConfiguration.`zonal-configuration`,
+  "zonalConfiguration"
+);
+@@clientName(RedisCommonPropertiesRedisConfiguration.`storage-subscription-id`,
+  "storageSubscriptionId"
+);
+@@clientName(RedisCommonPropertiesRedisConfiguration.`aad-enabled`,
+  "aadEnabled"
+);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisUpdateParameters.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisLinkedServerCreateParameters.properties);
+
+@@clientName(RedisResources.create::parameters.properties, "parameters");
+@@clientName(RedisResources.update::parameters.properties, "parameters");
+@@clientName(RedisResources.regenerateKey::parameters.body, "parameters");
+@@clientName(RedisResources.forceReboot::parameters.body, "parameters");
+@@clientName(RedisResources.importData::parameters.body, "parameters");
+@@clientName(RedisResources.exportData::parameters.body, "parameters");
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisResource.properties);
+
+@@clientName(PrivateEndpointConnections.put::parameters.resource, "properties");
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(PrivateEndpointConnection.properties);
+
+@@clientName(RedisFirewallRules.createOrUpdate::parameters.resource,
+  "parameters"
+);
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisFirewallRule.properties);
+
+@@clientName(RedisPatchSchedules.createOrUpdate::parameters.resource,
+  "parameters"
+);
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisPatchSchedule.properties);
+
+@@clientName(LinkedServer.create::parameters.properties, "parameters");
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisLinkedServerWithProperties.properties);
+
+@@clientName(RedisCacheAccessPolicies.createUpdate::parameters.resource,
+  "parameters"
+);
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisCacheAccessPolicy.properties);
+
+@@clientName(RedisCacheAccessPolicyAssignments.createUpdate::parameters.resource,
+  "parameters"
+);
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(RedisCacheAccessPolicyAssignment.properties);
+@@clientName(ScheduleEntries.scheduleEntries, "ScheduleEntries");
+@@clientName(RedisOperationGroup.checkNameAvailability::parameters.body,
+  "parameters"
+);
```

Comment:
This is all for sake of SDK back compat right? OK I guess.

Reply:
Yes